set-meta-releases/action.yml

```
@@ -0,0 +1,38 @@
+name: Get Meta Releases
+description: |
+  Fetch the list of active meta releases.
+
+  The meta releases will be made available as a JSON list in the output
+  `meta-releases`.
+
+outputs:
+  meta-releases:
+    description: 'JSON list of meta release identifiers'
+    value: ${{ steps.set-meta-releases.outputs.meta-releases }}
+
+runs:
+  using: composite
+  steps:
+    - name: Configure Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+
+    - name: fetch branch file
+      shell: bash
+      run: |
+        wget https://raw.githubusercontent.com/cylc/cylc-admin/master/docs/status/branches.json
+
+    - id: set-meta-releases
+      shell: python
+      run: |
+        import json
+        import os
+        with open(os.environ['GITHUB_OUTPUT'], 'w+') as outputsfile:
+          with open('branches.json', 'r') as jsonfile:
+            branches = json.load(jsonfile)
+            print(
+            'meta-releases='
+              + json.dumps(list(branches['meta_releases'].keys())),
+              file=outputsfile,
+          )
```

Indentation is off. Might I suggest
```suggestion
            outputsfile.write(
              'meta-releases='
              f'{json.dumps(list(branches['meta_releases'].keys()))}'
            )
```

---

(note file.write does not add a newline at the end)

```suggestion
            print(
              'meta-releases='
              + json.dumps(list(branches['meta_releases'].keys())),
              file=outputsfile,
            )
```
```

---

Amusingly/confusingly, it did work like this.